superchain/configs/goerli-dev-0/base-devnet-0.yaml

```diff
@@ -7,6 +7,8 @@ explorer: ""
 system_config_addr: "0x4f775d578e3ab8ce81f5ec065050938dbd5fb8c2"
 batch_inbox_addr: "0x1176307100000000000000000000000000000000"
 
+canyon_time: 1698436800
```

Comment:
Why do we have to repeat the exact same timestamp in 4 configs of the same superchain target? Can we not make the configuration part of the `superchain.yaml`? (We can add an override option to the chain, but ideally we keep superchain member chain on the exact same hardfork schedule to avoid confusing inconsistencies)

Reply:
In op-node and op-geth the superchain-config is readily available when constructing the resulting rollup config and EL chain-config, so the superchain attribute can be pulled in that way, without duplication.

Reply:
Specifically addressed in the PR description.

> I do want to switch over to defining the canyon time in the superchain file, but it was easier to define it in each chain file for the moment. For consumers, there is no difference.

Reply:
Why was it easier? I'm curious, I thought that by default we want to add activation times into the `superchain.yaml`.

Reply:
I don't see why we cannot just change it to the superchain config already, I disagree with the PR description on what's easier there.

Reply:
I went with this approach b/c I tend to think of HF as per chain (i.e. look up a HF by chain ID). Looking more closely, I see that when we actually pull the chain config per chain, we also pull the SuperchainConfig. I could load the value directly from that. I was initially thinking that I'd have to set the value in the per-chain config from the superchain config at init time.

In general, there's no documentation on what structs are important/what they do which made it hard to find how to do it with the supchain-config